doc/version_history.rst

```diff
@@ -2,6 +2,11 @@
 Version History
 ===============
 
+v0.8.1
```

Comment:
Yup, this is definitely a patch release, no code change at all besides a test fix.